src/components/NotFound/NotFoundComponent.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import { StyledNotFound } from 'components/NotFound/NotFoundStyles';
+
+export function NotFound() {
+  return (
+    <StyledNotFound>
+      <div className="not-found">
+        {/* eslint-disable-next-line react/no-unescaped-entities */}
```

Comment:
Ignoring eslint here doesn't look like a great solution :)
Let's better do what they recommend on stackoverflow :)
https://stackoverflow.com/a/45898131/1657101

The same advice is given on the [eslint rule page](https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/no-unescaped-entities.md):
![image](https://user-images.githubusercontent.com/6686679/115617563-32a30400-a2fa-11eb-82a3-7f7e5f49eaf9.png)

Or the following might work as well:
```
<h3>{'We didn\'t find any matches'}</h3>
```